index.html

```diff
@@ -89,7 +89,7 @@ <h5> Exotics </h5>
         <img src="https://mershark.github.io/images/icons8-lecture-64.png" />
         <h5 >Electric cars</h5>
         <p >The fun begins when you're on the road. 
-          You'll instantly feel a difference in the ride the first time you accelerate from a dead stop.</p>
+          You'll instantly feel a difference in the ride the first time you accelerate from a dead stop</p>
       </li>
 
```

Comment:
- [ ] Kindly make sure all the `<img>` tags has an `alt` attribute so they are accessible, these images doesn't have `alt` tags

```
<img src="https://mershark.github.io/images/icons8-lecture-64.png" />
```